errors.go

```diff
@@ -149,8 +149,27 @@ func defaultErrorHandler(status int, origErr error, c Context) error {
 	c.Response().WriteHeader(status)
 
 	if env != nil && env.(string) == "production" {
-		responseBody := productionErrorResponseFor(status)
-		c.Response().Write(responseBody)
+		switch strings.ToLower(ct) {
+		case "application/json", "text/json", "json":
+			err := json.NewEncoder(c.Response()).Encode(&ErrorResponse{
+				Error: errors.Cause(origErr).Error(),
+				Code:  status,
+			})
+			if err != nil {
+				return errors.WithStack(err)
```

Comment:
`WithStack` shouldn't be part of the response. Imagine bein a consumer of an API that is used in production and it tells you a lot of unneeded Trace. Let it aside that it could be even dangerous to leak such information to exposed world.

Reply:
Is it? Wouldn't it just call the default error handler another time, just overwriting the initial error and not leaking the trace since it's in production? Do I miss anything here?
But I guess, since we wouldn't use the added trace, we could also just omit it and just return the new err.